src/UNIV2LPOracle.t.sol

```diff
@@ -546,6 +549,33 @@ contract UNIV2LPOracleTest is DSTest {
         assertTrue(daiEthLPOracle.pass());                           // Fail pass
     }
 
+    // This test examines the gas costs of a common pattern which needs
+    // to be maximally gas-optimized.
+    function test_gas_pass_poke() public {
+        uint256 preGas;
+        uint256 diffGas;
+
+        require(daiEthLPOracle.pass());
+
+        preGas = gasleft();
+        if (daiEthLPOracle.pass()) {
```

Comment:
You were probably trying to mimic the pattern of keepers here, but it just makes the test harder to comprehend without adding much. I would remove these two conditionals.

Reply:
I wanted to be as close as possible to the "real" code that we want to minimize the cost of, on the off chance that this affected anything. But OTOH this repo probably doesn't even use the same compiler or optimization settings as the `MegaPoker`, so might as well simplify.